manifest.py

```diff
@@ -234,3 +234,70 @@ def create_manifest(sources, tileset, license, account, product, date, notes,
 if __name__ == "__main__":
     create_manifest()
 
+# PyTest only below this line
+
+from click.testing import CliRunner
+from jsonschema import FormatChecker
+from jsonschema import validate
+import pytest
+
+import json
+
+
+@pytest.fixture(scope='function')
+def manifest_args():
+    """Fixture for manifest args recreated on each function call."""
+    return ['-t', 'accountname.tileset',
+            '--license', '"CC BY-SA"',
+            '--account', 'accountname',
+            '--product', 'productname',
+            '--date', '2018'
+            ]
```

Comment:
I think drawing a conceptual line between fixtures (temporary files and directories, mock services, &c) and function or command parameters will help us write better tests. And in this particular case, since there's nothing to be torn down, fixture is overkill. A module constant is the simplest thing that will work.

Reply:
@sgillies having this as a fixture with `@pytest.fixture(scope='function')` is useful, it allows the manifest args to be modified in each test but recreated before the next function call. An example is in https://github.com/mapbox/pxm-manifest-specification/blob/manifest_testing/manifest.py#L280